pkg/api/common/helper.go

```diff
@@ -466,6 +466,9 @@ version = 2
           runtime_type = "io.containerd.runc.v2"
         [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.untrusted]
           runtime_type = "io.containerd.runc.v2"
+    [plugins."io.containerd.grpc.v1.cri".registry]
```

Comment:
@marosset @jsturtevant where does the equivalent Windows containerd config live?

Reply:
https://github.com/Azure/aks-engine/blob/master/parts/k8s/containerdtemplate.toml